### PR TITLE
Fix issue with +54 rewards

### PR DESCRIPTION
**Pull request by DoctaEnkoda**

### CODE REVIEW

File: src/main/java/com/iridium/iridiumskyblock/gui/IslandRewardsGUI.java [hunk 1]
```diff
@@ -27,11 +27,14 @@ public void addContent(Inventory inventory) {
...
-
+        
+        int maxSize = inventory.getSize() - 1; // slot 0
+        
         AtomicInteger atomicInteger = new AtomicInteger(0);
-        IridiumSkyblock.getInstance().getDatabaseManager().getIslandRewardTableManager().getEntries(getIsland()).forEach(islandReward ->
-                inventory.setItem(atomicInteger.getAndIncrement(), ItemStackUtils.makeItem(islandReward.getReward().item, placeholders))
-        );
+        for (IslandReward islandReward : IridiumSkyblock.getInstance().getDatabaseManager().getIslandRewardTableManager().getEntries(getIsland())) {
+            if (atomicInteger.get() > maxSize) continue;
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/gui/IslandRewardsGUI.java:35]: You can just break, can't you?

Comment (DoctaEnkoda) [reply src/main/java/com/iridium/iridiumskyblock/gui/IslandRewardsGUI.java]: yes mb

**Review by dlsf — APPROVED** [review]

**Comment by PeachesMLG** [comment]
Good spot, if we are adding this, I think we should add next page / previous page buttons aswell

**Review by PeachesMLG — CHANGES_REQUESTED** [review]
Good spot, if we are adding this, I think we should add next page / previous page buttons aswell

**Comment by dlsf** [comment]
I don't really feel like that's needed, rewards just appear after you claim them. Why would you want to claim a specific reward?

**Comment by PeachesMLG** [comment]
idk, people are weird

**Comment by dlsf** [comment]
@DoctaEnkoda are you planning on adding pagination support or should someone else do it?

**Comment by DoctaEnkoda** [comment]
Frankly, the pagination is useless because we only recover. Why weigh down something that doesn't need it.

**Pull request merged**